releases/v1/ccip_receiver.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "CcipReceiver",
+    "version": "1",
```

Comment:
Is the version format going to be 1, 2, 3... or 1.0, 1.1, 2.0...?

Reply:
we're going with int instead of string, right @ykim  ?